### ISSUE TITLE
Should <img> support role=figure?

### ISSUE THREAD

**Issue by cyns** [w3c/html-aria#319]
The definition of figure is one or more images, but images can be quite complex internally. One use case would be SVG images with sub-parts and text inside. Figure is a kind of image.

**Comment by stevefaulkner** [comment]
Hi @cyns welcome back!
>The definition of figure is one or more images

this is not correct, the [definition of `figure` in ARIA](https://www.w3.org/TR/wai-aria-1.2/#figure) is:
> A perceivable section of content that typically contains a graphical document, images, code snippets, or example text... 

[in HTML](https://html.spec.whatwg.org/multipage/grouping-content.html#the-figure-element)
> The figure element represents some flow content, optionally with a caption...

**Comment by scottaohara** [comment]
I spent some time kicking this idea around and the [results were not particularly good](https://scottaohara.github.io/testing/figure/img-figure.html).

tldr; aside for a few instances, placing a `role=figure` on an `<img>` caused the image to become completely hidden from screen readers.

Regarding the complex SVG scenario, even if `role=figure` on an image had better support, this wouldn't be a viable use case as screen readers can't enter the SVG's DOM when its a source of an `<img>` element.  However, this can work just fine for an actual `<svg>`, and that element does allow any role.

Per the above, I'm going to close this issue as I don't want to allow something that'd make the content inaccessible.
